doc/meta: make meta.description consistent with contributing document (#324352)
NixOS/nixpkgs: doc/stdenv/meta.chapter.md

## Standard meta-attributes {#sec-standard-meta-attributes}

> If the package is to submitted to Nixpkgs, also check out
> [requirements of meta attributes](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes)
> in the contributing documentation.

Reviewer: The current wording sounds a bit off to my ear, I could be wrong, yet this sounds more correct to me:
```suggestion
If the package is to be submitted to Nixpkgs, please check out the
[requirements for meta attributes](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes)
in the contributing documentation.
```
